packages/create-sitecore-jss/src/templates/nextjs-personalize/.env

```diff
@@ -4,6 +4,10 @@ NEXT_PUBLIC_CDP_TARGET_URL=
 # Your Sitecore CDP client key
 NEXT_PUBLIC_CDP_CLIENT_KEY=
 
+# Your optional Sitecore Scope unique identifier allowing you
+# to isolate your personalization data between XM Cloud environments
```

Comment:
Suggested comment
```suggestion
# An optional Sitecore Personalize scope identifier.
# This can be used to isolate personalization data when multiple XM Cloud Environments share a Personalize tenant.
# This should match the PAGES_PERSONALIZE_SCOPE environment variable for your connected XM Cloud Environment.
```